lib/Doctrine/ORM/QueryBuilder.php

```diff
@@ -1506,6 +1514,74 @@ public function __toString()
         return $this->getDQL();
     }
 
+    /**
+     * Creates a new named parameter and bind the value $value to it.
+     *
+     * This method provides a shortcut for {@see Statement::bindValue()}
+     * when using prepared statements.
+     *
+     * The parameter $value specifies the value that you want to bind. If
+     * $placeholder is not provided createNamedParameter() will automatically
+     * create a placeholder for you. An automatic placeholder will be of the
+     * name ':dcValue1', ':dcValue2' etc.
+     *
+     * Example:
+     * <code>
+     * $value = 2;
+     * $q->eq( 'id', $q->createNamedParameter( $value ) );
+     * $stmt = $q->executeQuery(); // executed with 'id = 2'
+     * </code>
+     *
+     * @link http://www.zetacomponents.org
+     *
+     * @param mixed           $value
+     * @param int|string|null $type        ParameterType::* or \Doctrine\DBAL\Types\Type::* constant
+     * @param string          $placeHolder The name to bind with. The string must start with a colon ':'.
+     *
+     * @return string the placeholder name used.
+     */
+    public function createNamedParameter($value, $type = ParameterType::STRING, $placeHolder = null)
```

Comment:
Please use native parameter and return types when possible.

Reply:
Hi,
i rebased to 2.13.x and changed the default parameter value.